sql/core/src/test/resources/sql-tests/results/describe.sql.out

```diff
@@ -530,7 +530,7 @@ struct<plan:string>
 -- !query output
 == Physical Plan ==
 Execute DescribeTableCommand
-   +- DescribeTableCommand `t`, true
+   +- DescribeTableCommand `default`.`t`, true
```

Comment:
This is the consequence of removing the hack. Now `ResolvedTable` can get qualified table name even from session catalog.